Header section

WiLexa opened the pull request.

WiLexa committed:
Merge pull request #4 from marianamasia/main

hidden

WiLexa committed:
work

andriiKrakovskyi committed:
Merge branch 'main' into header-section

andriiKrakovskyi commented on src/css/header.css:
```diff
@@ -1,37 +1,74 @@
+  justify-content: space-between;
+  max-width: 375px;
+  padding-left: 20px;
+  padding-right: 20px;
```
хедер должен заходить на секцию hero

    position: absolute;
    top: 20px;
    left: 20px;
    right: 20px;
    z-index: 99999999999;

andriiKrakovskyi commented on src/css/header.css:
```diff
@@ -1,37 +1,74 @@
+  display: none;
+}
+.header-nav-list {
+}
```
нельзя оставлять пустые классы

WiLexa committed:
Mob version

WiLexa committed:
all

Я зробив адаптивку на всі пристрої, але вкінці вже не працювала моб. версія. Будьласка, скажіть що тут зламалось? я вже не думаю. я завтра зранку та після роботи (22-00) виправлю.

andriiKrakovskyi committed:
Merge branch 'main' into header-section
